scripts/apps/search/views/multi-image-edit.html

```diff
@@ -0,0 +1,128 @@
+            <div class="sd-photo-preview sd-photo-preview--grid-only">
+                <div class="sd-photo-preview__thumb-strip">
+                    <div class="sd-grid-list dark-ui">
+
+                        <div class="sd-grid-item sd-grid-item--with-click"
+                             ng-click="selectImage(image)"
+                             ng-repeat="image in origin track by image._id"
+                             ng-class="{'sd-grid-item--selected' : !image.unselected}">
+                            <div class="sd-grid-item__thumb">
+                                <img ng-src="{{image.renditions.thumbnail.href}}"/>
+                            </div>
+                            <div class="sd-grid-item__content">
+                                <time sd-absdate datetime="image.versioncreated"></time>
+                                <p class="sd-grid-item__title">{{image.headline}}</p>
+                                <p class="sd-grid-item--element-grow sd-grid-item__text-light">{{image.description_text}}</p>
+                                <div class="sd-grid-item__content-block">
+                                    <span class="sd-grid-item__text-label">Copyright:</span>
+                                    <span class="sd-grid-item__text-strong">{{image.copyrightnotice}}</span>
+                                </div>
+                            </div>
+                            <div class="sd-grid-item__footer">
+                                <div class="sd-grid-item__footer-block sd-grid-item__footer-block--multi-l">
+                                    <i class="icon-photo sd-grid-item__type-icn sd-grid-item__footer-block-item"></i>
+                                </div>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="sd-column-box__slide-in-column sd-column-box__slide-in-column--large sd-column-box__slide-in-column--dark-ui">
+            <div class="sd-column-box__slide-in-column-inner sd-slide-in-panel">
+                <div class="sd-slide-in-panel__header">
+                    <h3 class="sd-slide-in-panel__heading" translate>Metadata</h3>
+                </div>
+                <div class="sd-slide-in-panel__content">
+                    <div class="sd-slide-in-panel__content-block sd-slide-in-panel__content-block--first">
+                        <div class="sd-alert sd-alert--hollow sd-alert--primary sd-alert--small" style="margin: -1rem 0 3rem;">
+                            <span translate>Currently editing: {{images.length}} items</span>
+                        </div>
+                        <div class="form__row" ng-if="validator.headline">
+                            <div class="sd-line-input sd-line-input--dark-ui sd-line-input--boxed" ng-class="{'sd-line-input--required':validator.headline.required}">
```

Comment:
imo it could use a directive for line input, there is so much boilerplate html, it could be like:
```hml
<div sd-line-input
    data-dark="true"
    data-boxed="true"
    data-required="{{ validator.headline.required }}"
    data-label="{{ :: 'Title' | translate }}"
    data-model="{{ metadata.headline }}"
    data-change="..."
    ...
></div>
```
this way the forms might get eventually consistent with all the ui framework classes being set on a few directives